_data/projects/SuperAGI.yml

```
@@ -0,0 +1,16 @@
+name: SuperAGI
+desc: An open-source framework for developers to Build, manage & run Autonomous AI agents
+site: https://github.com/TransformerOptimus/SuperAGI
+tags:
+  - agi
+  - python
+  - ai
+  - autonomous-agents
+  - nextjs
+upforgrabs:
+  name: help wanted
+  link: https://github.com/TransformerOptimus/SuperAGI/issues
```

This should be a link to a subset of issues that are ideal for new contributors. I see you have the `help wanted` label but there are no issues assigned to that label.

If you do end up labelling some issues this should do the trick:

```suggestion
  link: https://github.com/TransformerOptimus/SuperAGI/labels/help%20wanted
```